src/InverseSolve/iteratedTikhonov.jl

```diff
@@ -0,0 +1,51 @@
+export iteratedTikhonov
+
+function iteratedTikhonov(m0,pInv::InverseParam,PF::Array{RemoteRef{Channel{Any}}},nBeta,betaFac,targetMisfit;indCredit=[],dumpResults::Function = dummy)
+
+  mc = m0
+  dObs = Float64[]
+  Wd   = Float64[]
+  for i=1:length(pInv.dobs)
+    dObs = [dObs;vec(fetch(pInv.dobs[i]))]
+    Wd = [Wd;vec(fetch(pInv.Wd[i]))]
+  end
+  alphaInit = pInv.alpha
+  iter = 0
+  DcRef = Array{RemoteRef{Channel{Any}},length(PF)}
+  while iter < nBeta 
+    iter += 1
+    pInv.alpha = pInv.alpha/betaFac
+    mc,DcRef,flag,hist = projGNCG(mc,pInv,PF,dumpResults=dumpResults)
+    pInv.mref = mc
+    DcLoc = Float64[]
+    for i=1:length(DcRef)
+      DcLoc = [DcLoc;vec(fetch(DcRef[i]))]
+    end
+    F,dF,d2F = pInv.misfit(DcLoc,dObs,Wd)
```

Comment:
the current misfit ```F``` should be stored in ```hist```. If you removed this line here, it seems to me that you don't need to add two methods for ```iteratedTikhonov```.